data/json/items/gun/12mm.json

```diff
@@ -4,8 +4,8 @@
     "copy-from": "gun_base",
     "looks_like": "ar15",
     "type": "GUN",
-    "name": "H&K G80 Railgun",
-    "description": "Developed by Heckler & Koch in the first quarter of 21st century, the railgun magnetically propels a ferromagnetic projectile using an alternating current.  Powered by UPS.",
+    "name": "H&K G80 railgun",
+    "description": "Developed by Heckler & Koch, this railgun magnetically propels a ferromagnetic projectile using an alternating current.  Powered by UPS.",
```

Comment:
Current year is technically in the first quarter of the 21st century.

Reply:
It's probably not the best way to write it like that until current game year is at least 2026. That's just my opinion though; could revert it if it's okay.